membership-attribute-service/app/services/GuardianPatronService.scala

```diff
@@ -83,7 +83,7 @@ class GuardianPatronService(
           RatePlan(
             id = RatePlanId("guardian_patron_unused"), // only used for contribution amount change
             productRatePlanId = Catalog.guardianPatronProductRatePlanId,
-            productName = "Guardian Patron",
+            productName = "guardianpatron",
```

Comment:
this was misaligned with what we expect to appear in the tier property so I have updated to match.  (Although I would probably have gone for the former, it looks like manage relies on the latter in many places already.)